.github/workflows/process-images.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@master
+        uses: actions/checkout@main
       - name: Compress Images
-        uses: calibreapp/image-actions@master
+        uses: calibreapp/image-actions@main
```

Comment:
Does calibreapp use a `main` branch? Just wasn't sure 👀

Reply:
oh that's a good question lol I guess I should check! I suppose maybe not, since we've been using master for the config and that hasn't broken lololol 😅

Reply:
yeah, they're on master. I'll update this. Good catch!